modules/terraform-zscc-lb-azure/main.tf

```diff
@@ -43,12 +44,11 @@ resource "azurerm_lb_backend_address_pool" "cc_lb_backend_pool" {
 # Define load balancer health probe parameters
 ################################################################################
 resource "azurerm_lb_probe" "cc_lb_probe" {
-  name                = "${var.name_prefix}-cc-lb-probe-${var.resource_tag}"
-  resource_group_name = var.resource_group

```

Comment:
Was it complaining when you had resouce_group_name?  This is not applicable I believe and hence you removed it.  How did you figure this out?